### ISSUE TITLE
The board picks up smartphone communication signals

### ISSUE THREAD

**Issue by jzli559** [sinara-hw/Urukul#85]
I observed that the Urukul board can pick up signals from smartphone communications. Using an oscilloscope, I probed the signal from the SMA connector on the front panel of the board. Throughout the process, I didn't set any output of the board. When I made a voice or video call on a smartphone and brought the phone close to the SMA connector, I observed high-frequency (over 1 GHz) signals on the oscilloscope. After turning off the call, the signal returned near zero.

I did the test under different configurations, and the results were similar.
- Urukul installed in a crate: connected to a Kasli that is powered and grounded.
- Urukul installed in a crate: connected to a Kasli that is not powered but the crate is grounded.
- Urukul installed in a crate: connected to a Kasli, not powered and not grounded.
- Urukul isolated: not connected, not powered, and not grounded.

I did further tests with a spectrum analyzer (FSV3030), and the results are as follows:

In crate, grounded, not powered; no phone:
![FSV3000_ScreenShot_2024-12-18_10-20-55](https://github.com/user-attachments/assets/52c534ce-de76-4807-8f20-b5e657129bf1)

In crate, grounded, not powered; with phone (5 cm distance):
![FSV3000_ScreenShot_2024-12-18_10-16-03](https://github.com/user-attachments/assets/3d79c84a-187d-41f8-a59d-523826dfcf2d)

In crate, not grounded, not powered; no phone:
![FSV3000_ScreenShot_2024-12-18_09-40-02](https://github.com/user-attachments/assets/d89eb425-f47a-4589-be32-9eca18fbadb8)

In crate, not grounded, not powered; with phone (5 cm distance):
![FSV3000_ScreenShot_2024-12-18_10-14-10](https://github.com/user-attachments/assets/05507482-0d67-401a-9f72-b9fd7e246025)

After the tests above, I additionally tested the noise by setting the channel to output a signal (100 MHz, 0 dB att, 1.0 amp).

100 MHz, 0 dB att, 1.0 amp, no phone:
![FSV3000_ScreenShot_2024-12-18_09-38-55](https://github.com/user-attachments/assets/9015b94a-b7a4-4290-8731-06d69141ee6f)

100 MHz, 0 dB att, 1.0 amp, with phone (5 cm distance):
![FSV3000_ScreenShot_2024-12-18_09-38-35](https://github.com/user-attachments/assets/887c53b6-c895-4e4f-a0ad-de33a43480c9)

For simplicity, all six images above were tested on channel 0. The only related configurations are the ground and the phone. The results of others are similar. Some of the noise peaks were related to the ground, but M4 and M8 were clearly related to the phone. The two peaks did not return to the normal level until the distance was increased to about 3 meters.

This looks like the board doesn't specifically handle noise over 1 GHz, and the wavelengths of some of the phone signals are similar to the length of some wires on the board.

Similar phenomena were observed when testing other cards in the same crate, including the Kasli J3, Clocker Output 9, and all channels of the DIO-BNC card. Other interfaces haven't been tested yet.

**Comment by gkasprow** [comment]
pls do the following test:
- use the same RF cable. 
- place 50Ohm termination and then short SMA nut with unpowered, grounded crate. Beware of alu oxidation layer - make sure you attach the SMA terminator to some steel screw/
- measure the spectrum with the phone as above
- do not place 50Ohm termination and then short SMA nut with unpowered, grounded crate
- measure spectrum with the phone as above

**Comment by jzli559** [comment]
Before anything else, I need to correct the test results I posted earlier. Both the first and second images were powered and grounded, with no DDS output set.

Based on the suggestion, I did some new tests. First, the spectrum analyzer (SA) requires grounded during using, and it shares the same ground with the crate. Therefore, the SMA nut of the RF cable is inherently grounded with the crate without being shorted again. I'm not sure if connecting the RF cable from the SA to the Urukul's SMA would create a ground loop.

Below are the new test results.

(Remeasured) In crate, grounded, not powered; no phone:
![FSV3000_ScreenShot_2024-12-21_07-10-32](https://github.com/user-attachments/assets/3c4117b1-d021-47f5-a55c-844676960aae)

(Remeasured) In crate, grounded, not powered; with phone (5 cm distance):
![FSV3000_ScreenShot_2024-12-21_07-07-14](https://github.com/user-attachments/assets/f43436ab-cc37-481b-ab86-08fe7723130f)
![FSV3000_ScreenShot_2024-12-21_08-34-56](https://github.com/user-attachments/assets/85675c43-96e6-4457-ad5c-70db74247e1b)
![FSV3000_ScreenShot_2024-12-21_08-35-34](https://github.com/user-attachments/assets/8e331f20-459f-44dc-83ad-d3cf97b05d89)

All the following tests are also within crate, grounded, not powered.

Urukul ch0 connected to a 50 Ohm terminator; SA probe connected to noting or a 50 Ohm; phone close to ch0:
almost no signal.

Urukul ch0 connected to nothing; SA probe connected to noting or a 50 Ohm; phone close to ch0:
almost no signal.

Urukul ch0 connected to nothing or a 50 Ohm; SA probe connected to noting; phone close to SA probe:
almost no signal.

Urukul ch0 connected to nothing; SA probe connected to a 50 Ohm; phone close to SA probe:
(Urukul ch0 connected to a 50 Ohm was similar)
![FSV3000_ScreenShot_2024-12-21_08-12-17](https://github.com/user-attachments/assets/1af556f5-2cf9-4d47-8ed4-2b5ef3c3ae87)

The last image also showed some clear noise signals but not as big as those from the probe connected to ch0.

**Comment by gkasprow** [comment]
Is the front panel grounded?

**Comment by jzli559** [comment]
> Is the front panel grounded?

Yes. The card is installed in the crate. I measured with a multimeter and found that the outer shell of the SMA connector on the front panel is electrically connected to the crate.

**Comment by gkasprow** [comment]
That's not enough. You checked DC path but we are interested in AC path. Just check if the grounding resistors are soldered.

**Comment by jzli559** [comment]
Do you mean using the multimeter to test resistors R179 ~ R182?

**Comment by gkasprow** [comment]
There are 2 resistors amd 2 capacitors that comnect panel with PCB ground
